mysql-test/include/gap_lock_error_cleanup.inc

```diff
@@ -0,0 +1,6 @@
+
+drop table if exists gap1, gap2, gap3, gap4;
+
+--exec grep "Got error" $MYSQLTEST_VARDIR/log/mysqld.1.err | grep "gap" | wc -l
```

Comment:
Actually, this is a really good catch. I'm not sure why this is looking into the err log as there is nowhere that anything is ever written to the server error log on a gap lock violation, all errors go to the client and in the upstream patch they are also optionally written to the gap lock log. I believe this serves no useful purpose at all.